core/src/main/java/org/apache/iceberg/CatalogUtil.java

```diff
@@ -341,4 +341,15 @@ public static void configureHadoopConf(Object maybeConfigurable, Object conf) {
 
     setConf.invoke(conf);
   }
+
+  public static String cleanWarehousePath(String path) {
```

Comment:
`LocationProviders` already exposes `stripTrailingSlash`, which removes multiple trailing slashes. Could you use that instead?

Reply:
sure done :) !!